### PR TITLE
site: add ffda-ssh-manager

### PR DESCRIPTION
**Pull request by blocktrron**
Add configuration and package for ffda-ssh-manager.

This will allow us to roll out SSH keys on nodes where ssh-manager was enabled on.

Note that this is a manual process and requires the user *actively* setting the uci flag or compiling firmware where ssh-manager is default enabled.

### CODE REVIEW

File: site.conf [hunk 1]
```diff
@@ -19,6 +19,20 @@
...
+	ssh_manager = {
+		defaults = {
+			enabled = false,
+			groups = {
+				-- No default groups
+			},
+		},
+		groups = {
+			refugees = {
+				'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDM6d1E1gczThXTfwGLuzbo9t3xUdpp53wi9Ya5IyxUlB0rNO9Nc2mKvI99Pp/4POan6vAj+dyEA9ve5WtYfOKYrwHHKnUsuSxDMORYhnGjuzrjM5hdJvVgtfmGV9ilC419Jy3+nxY2jZ992hCd/IP8i5epIoZIDDJ6QaSAVKIt3YZWpKxzalty5ugBlknkvaX5q7n32qIVx6cxpNAPjXVJ/rk1tzKldXkNCL36HjHZhPCqwzOVpLbjrGfEonFuU1sSR+8UfzlD7JeOSrwUck6MGEGFvCJVG/NqfNTe3tlMfc4bV69U9b4bsLeCVDXNAaxqByZoNrs/7jAmRdBeJBnylhvKdTb/lF3Nhs/Vor4H/ih+XL1suzu0z4lC1MiAuZrgBL+vQmLGpbjaKD1Px7awQuTrQ8Y6Faed76L0a22bzx/+2yN32VJ5if7cjg72gCzffHwZkjU8tnFuNAGGMTz6EQCljFMwJQTjYaxKz+bp559jO2Av5WlWNeasB2qZP8m/IRDTUt7jpcS9EZUWt66dTP6Xt+m2rXlyzREsekGiYp+4Ew41LNmAme+jhHUb9NlvtuxNxD555IzscSjwxVraNhfiaQbh6mGz3HM6fPYe7R7C8An2xo+FW5W2mq9n2OnmUDjx9pGIn24c2ji8oT9m9lqclXI1tbP9Nw5b1aKp8Q== blocktrron_freifunk',
```
Comment (mweinelt) [inline site.conf:44]: eddsa when?

File: image-customization.lua [hunk 2]
```diff
@@ -22,6 +22,7 @@ features({
 packages({
 	'iwinfo',
 	'respondd-module-airtime',
+	'ffda-ssh-manager',
```
Comment (herbetom) [inline image-customization.lua:25]: The package actually seems to be called `ffda-gluon-ssh-manager`?

https://github.com/freifunk-gluon/community-packages/tree/master/ffda-gluon-ssh-manager

Comment (blocktrron) [reply image-customization.lua:25]: No. https://github.com/freifunk-gluon/community-packages/blob/master/ffda-gluon-ssh-manager/Makefile#L21

File: site.conf (hunk 1 above)
Comment (blocktrron) [reply site.conf:44]: Christmas and Easter occur on the same date

File: image-customization.lua (hunk 2 above)
Comment (herbetom) [reply image-customization.lua:25]: okay, wasn't aware that dir and package name can be different. Would've expected that it's part of the Search path.
I would prefer if they were identical (less confusion), but i guess it's okay if it works.

**Pull request merged**